packages/onboarding/src/setup-tailored-site-after-creation.ts

```diff
@@ -1,19 +1,14 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
-import {
-	Onboard,
-	Site,
-	OnboardSelect,
-	SiteActions,
-	updateLaunchpadSettings,
-} from '@automattic/data-stores';
-import { select, dispatch } from '@wordpress/data';
+import { Onboard, OnboardSelect, Site } from '@automattic/data-stores';
+import { select } from '@wordpress/data';
 import wpcomRequest from 'wpcom-proxy-request';
-import { isNewsletterFlow, isFreeFlow } from './utils';
-import type { ActiveTheme } from '@automattic/data-stores';
+import { isNewsletterFlow } from './utils';
 
 const ONBOARD_STORE = Onboard.register();
+
+// TODO: This store registration should live somewhere else or just be called more centrally.
 // `client_id` and `client_secret` are only needed when signing up users.
-const SITE_STORE = Site.register( { client_id: '', client_secret: '' } );
+Site.register( { client_id: '', client_secret: '' } );
```

Comment:
Removing this altogether breaks some unrelated tests. I'm not sure what else it might break, since this package is extensively used, so I prefer keeping the store registration in place. This might also be solved once #74399 is done.

Reply:
> Removing this altogether breaks some unrelated tests

That's weird and unexpected 🤔 It may be worth a follow-up investigation.